### DISCUSSION TITLE
Possible to filter books by wordwise/x-ray status?

### DISCUSSION THREAD

**Discussion by tehguitarist** [xxyzz/WordDumb#221] [Q&A] [answered]
Is there a way to filter books by whether they have x-ray or wordwise files generated? I'm definitely missing having done some, would be nice to find out which ones without opening each folder and seeing if it has the files inside. I could probably devise a script that looks for the files and returns a list of directories that don't have those files, but a calibre based filter would be fun.

**Comment by xxyzz** [comment, marked as answer]
If the files are already created then the plugin won't recreate them so you don't really need to filter books.
